server.js

```diff
@@ -201,6 +201,20 @@ server.post('/invitation', jsonParser, function(req, res) {
   }
 })
 
+server.post('/activity_email', jsonParser, function(req, res) {    
+  var email = new sendgrid.Email({
+    to:       req.body.users,
+    from:     'wolontariat@krakow2016.com',
+    subject:  req.body.subject,
+    text:     req.body.text
+  })
+  sendgrid.send(email, function(err, json) {
+    console.log('sendgrid:', err, json)
+    res.send(err || json)
+  })  
+})
```

Comment:
Chciałbym wysłać maila na zdarzenia:
-utworzenie aktywności
-edycja aktywności
-usunięcie aktywności
O tych zdarzeniach dowiaduję się na podstawie wywołania odpowiednich metod w ActivityAdministration (create, update, remove). Czyli one są po stronie klienta?

To żeby wiedzieć, że te zdarzenia nastąpiły musiałbym się przenieść do obsługi bazy danych (rethinkdb/activities.js - metody create, update, delete) i stamtąd wywowyłać sendgrida? To już będzie po stronie serwera?